index.html

```diff
@@ -395,7 +439,10 @@ <h2 class="heading">Download Booking Dapp Now</h2>
 												Terms &amp; Conditions
 											</a>
 										</li>
-										
+										<li><a href="http://eraswaptoken.io/pdf/eraswap-terms-conditions.pdf" target="_blank"> Era Swap Terms &amp; Conditions</a></li>
+										<li><a href="http://eraswaptoken.io/pdf/era-swap-howey-test-letter-august7-2018.pdf" target="_blank">Howey Test</a></li>
+ 
+                                         <li><a href="http://eraswaptoken.io/pdf/eraswap_whitepaper.pdf" target="_blank">ES White Paper </a></li>
```

Comment:
I think at some point of time, we need to enforce `prettier` style checks on every PR so that code indentations are proper.